serverless/README.md

```diff
@@ -0,0 +1,38 @@
+## NFA - Serverless
+
+### Requirements
+
+This sub-project of NFAs requires Node 18.  Specifically, this has been tested with 18.13.0 so far.
+
+### Setup
+
+After cloning the repo, ensure you run `yarn` in the root directory. After that, `cd` into the `serverless` directory and alsy run `yarn`.
+
+If you are deploying, make sure you have your AWS credentials set to environment variables or have setup AWS credentials using the AWS CLI.
+
+Basically, these values need to be set:
+```
+export AWS_ACCESS_KEY_ID=value
+export AWS_SECRET_ACCESS_KEY=value
+export AWS_SESSION_TOKEN=value
```

Comment:
are we able to use a `.env` file?

Reply:
I don't think we can use `.env` for that.  Since it is `serverless` and not a `node` environment, it doesn't pick it up.  But I think the proper way is to use the AWS CLI as per these instructions: https://www.serverless.com/framework/docs/providers/aws/guide/credentials/.